crates/bevy_macro_utils/src/lib.rs

```diff
@@ -8,10 +8,10 @@ pub use attrs::*;
 pub use shape::*;
 pub use symbol::*;
 
-use proc_macro::TokenStream;
+use proc_macro::{TokenStream, TokenTree};
 use quote::{quote, quote_spanned};
-use std::{env, path::PathBuf};
-use syn::spanned::Spanned;
+use std::{collections::HashSet, env, path::PathBuf};
```

Comment:
We probably can't use `bevy_utils::HashSet` here, but we can use `rustc-hash`'s hasher.